Neos.Fusion/Classes/Core/ObjectTreeParser/FilePatternResolver.php

```diff
@@ -56,13 +57,14 @@ class FilePatternResolver
      * @return array|string[]
      * @throws Fusion\Exception
      */
-    public static function resolveFilesByPattern(string $filePattern, ?string $filePathForRelativeResolves = null, string $defaultFileEndForUnspecificGlobbing = '.fusion'): array
+    public static function resolveFilesByPattern(string $filePattern, ?string $filePathForRelativeResolves, string $defaultFileEndForUnspecificGlobbing): array
     {
-        $filePattern = trim($filePattern);
+        $filePattern = Files::getUnixStylePath(trim($filePattern));
         if ($filePattern === '') {
             throw new Fusion\Exception("cannot resolve empty pattern: '$filePattern'", 1636144288);
         }
-        if ($filePattern[0] === '/') {
+        $isAbsoluteWindowsPath = str_contains($filePattern, ':') && preg_match('`^[a-zA-Z]:/[^/]`', $filePattern) === 1;
```

Comment:
> str_contains($filePattern, ':') &&

premature performance optimization is the root of all evil?

... anyways ...

Reply:
😈